PhysicsTools/NanoAOD/python/nanoDQM_cff.py

```diff
@@ -66,6 +66,23 @@
 
 run2_miniAOD_80XLegacy.toModify(nanoDQM.vplots, IsoTrack = None)
 
+## no-change policy in run2_nanoAOD_106Xv1 (nanoAOD-v8)
+from Configuration.Eras.Modifier_run2_nanoAOD_106Xv1_cff import run2_nanoAOD_106Xv1
+from Configuration.Eras.Modifier_run2_nanoAOD_devel_cff import run2_nanoAOD_devel
+_fatjet_plots_nom = copy.deepcopy(nanoDQM.vplots.FatJet.plots)
+_fatjet_plots_106Xv1 = cms.VPSet()
+for plot in _fatjet_plots_nom:
+    if (plot.name.value() != "nConstituents"):
+        _fatjet_plots_106Xv1.append(plot)
+(run2_nanoAOD_106Xv1 & ~run2_nanoAOD_devel).toModify(nanoDQM.vplots.FatJet, plots = _fatjet_plots_106Xv1 )
+_sv_plots_nom = copy.deepcopy(nanoDQM.vplots.SV.plots)
+_sv_plots_106Xv1 = cms.VPSet()
+for plot in _sv_plots_nom:
+    if (plot.name.value() != "charge"):
+        _sv_plots_106Xv1.append(plot)
+(run2_nanoAOD_106Xv1 & ~run2_nanoAOD_devel).toModify(nanoDQM.vplots.SV, plots = _sv_plots_106Xv1 )
```

Comment:
I suspect this is still rebased on top of an old IB.
this part of the sv plots is unrelated to this PR and is already in the 10_6_X
https://github.com/cms-sw/cmssw/blob/CMSSW_10_6_X/PhysicsTools/NanoAOD/python/nanoDQM_cff.py#L105-L108

Reply:
Sorry! Idk what happened since I did the rebasing in the CMSSW_10_6_X_2021-05-17-2300, but I've removed the sv lines.